vmdb/app/controllers/provider_foreman_controller.rb

```diff
@@ -475,13 +508,13 @@ def configuration_profile_node(id, model)
     else
       options = {:model => "ConfiguredSystem"}
       options[:where_clause] = ["configuration_profile_id IN (?)", @configuration_profile_record.id]
+      options[:where_clause] =
+        ["configuration_manager_id IN (?) AND \
+          configuration_profile_id IS NULL", id] if @configuration_profile_record.id.nil?
```

Comment:
may be better to use `@configuration_profile_record.new_record?`
... or your idea - use a method `empty_profile?(@configuration_profile_record)`